src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/ShellRenderer.cs

```diff
@@ -161,19 +170,17 @@ void UpdateCurrentItem()
 
         void UpdateFlyoutIsPresented()
         {
-            if (_drawer != null)
-            {
-                _drawer.IsOpen = Element.FlyoutIsPresented;
-            }
+            _drawer.IsOpen = Element.FlyoutIsPresented;
```

Comment:
Why do not check null of _drawer? ShellSection.FlyoutIsPresented could be set by user even if there is no flyitems

Reply:
Fixed.